docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md

```diff
@@ -1,25 +1,28 @@
 ---
 title: Debug add-ins using developer tools on Windows 10
-description: ''
-ms.date: 07/01/2019
+description: 'Debug add-ins using Microsoft Edge developer tools on Windows 10'
+ms.date: 12/16/2019
 localization_priority: Priority
 ---
 
 # Debug add-ins using developer tools on Windows 10
 
 There are developer tools outside of IDEs available to help you debug your add-ins on Windows 10. These are useful when you need to investigate a problem while running your add-in outside the IDE.
 
-The tool that you use depends on whether the add-in is running in Microsoft Edge or Internet Explorer. This is determined by the version of Windows 10 and the version of Office that are installed on the computer. To determine which browser is being used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). 
-
+The tool that you use depends on whether the add-in is running in Microsoft Edge or Internet Explorer. This is determined by the version of Windows 10 and the version of Office that are installed on the computer. To determine which browser is being used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

Comment:
"running in Internet Explorer (IE) or the Edge browser."
also
"To determine which browser you're using on your development computer, see..."